workers/vita/Dockerfile

```diff
@@ -17,56 +17,77 @@ RUN sed 's/^deb \(.*\)/deb-src \1/' /etc/apt/sources.list \
 ENV VITASDK=/opt/vitasdk
 ENV PATH=$PATH:$VITASDK/bin
 
-ARG URL=https://github.com/vitasdk/autobuilds/releases/download/master-linux-v567/vitasdk-x86_64-linux-gnu-2017-10-29_09-48-44.tar.bz2
+#Install VitaSDK base
+ARG URL=https://github.com/vitasdk/autobuilds/releases/download/master-linux-v611/vitasdk-x86_64-linux-gnu-2017-12-01_14-01-36.tar.bz2
 
 RUN wget --progress=bar:force:noscroll -O - $URL \
 	| tar --no-same-owner --no-same-permissions -C /opt/ -xjf -
 
-# These insecure, unversioned downloads are not great, but OpenSSL and SDL2 are
-# not trivially buildable for this target so we use these packages for the
-# moment. At the time of the first build these corresponded to
-# https://github.com/vitasdk/packages/commit/b62cb68c349f8f389a4b4e57356c398c43f5531d
-# in case someone needs to rebuild them from source later
-RUN wget --progress=bar:force:noscroll -O - http://dl.vitasdk.org/openssl.tar.xz \
-	| tar --no-same-owner --no-same-permissions -C $VITASDK/arm-vita-eabi -xJf -
-RUN wget --progress=bar:force:noscroll -O - http://dl.vitasdk.org/curl.tar.xz \
-	| tar --no-same-owner --no-same-permissions -C $VITASDK/arm-vita-eabi -xJf -
-RUN wget --progress=bar:force:noscroll -O - http://dl.vitasdk.org/sdl2.tar.xz \
-	| tar --no-same-owner --no-same-permissions -C $VITASDK/arm-vita-eabi -xJf -
-RUN wget --progress=bar:force:noscroll -O - http://dl.vitasdk.org/sdl_net.tar.xz \
-	| tar --no-same-owner --no-same-permissions -C $VITASDK/arm-vita-eabi -xJf -
-
+#Install git
+RUN apt-get -y install git
+
+#Install sudo
+RUN apt-get -y install sudo
+
+#Install curl
+RUN apt-get -y install curl
+
+#Install Vita versions of most common libraries, including ogg, png, zlib etc.
+RUN git clone https://github.com/vitasdk/vdpm && \
+	cd vdpm && \
+	./install-all.sh && \
+	cd ..
+
+#Install recent SDL2 port for Vita
+RUN git clone https://github.com/rsn8887/SDL-Vita && \
+	cd SDL-Vita && \
+	make -f Makefile.vita install && \
+	cd ..
+
+#Install libmad for Vita
+RUN git clone https://github.com/Cpasjuste/libmad-psp2 && \
```

Comment:
There is no reason given for why any of this is being done. There has to be a good reason for why a derived library is used, and it must be documented. This is true for all system ops all the time (and in programming too), not just here.